workflow.md

```diff
@@ -0,0 +1,67 @@
+# Workflow
+
+### The Do's
+
+ - Divide work to be done into Sprints.
+ - Select [Sprint] length of either 1 or 2 weeks, max.
+ - Define type of Sprint as either a [Feature Sprint] or a [Housekeeping Sprint].
+ - Plan before each [Sprint].
+ - Own your [Pull Requests].
+ - Reflect after each [Sprint].
+ - Start Sprints on Tuesdays.
+ - Deploy on Mondays.
+ - Name branches appropriately.
+
+
+### The Don'ts
+
+ - **NO** Friday deployments.
+ - **NO** failing tests on master.
```

Comment:
I'd vote to have an additional **Don't** like this: `NO PR that you haven't QAd first yourself.`